integrationtest/spring/src/test/resources/beans.xml

```diff
@@ -70,7 +70,7 @@
                 <!-- 2 is TRANSACTION_READ_COMMITTED -->
                 <entry key="hibernate.connection.isolation" value="2"/>
                 <!-- http://docs.jboss.org/hibernate/core/3.3/reference/en/html/architecture.html#architecture-current-session -->
-                <entry key="hibernate.current_session_context_class" value="org.hibernate.context.JTASessionContext"/>
+                <entry key="hibernate.current_session_context_class" value="org.hibernate.context.internal.JTASessionContext"/>
```

Comment:
> Or is there a particular reason to reference the class explicitly?

Yes, me not knowing about the `jta` configuration option ;-)

Thanks, I'll change it.